dev-docs/bidders/showheroes-bs.md

```diff
@@ -19,6 +23,127 @@ sidebarType: 1
 {: .table .table-bordered .table-striped }
 | Name        | Scope                            | Description                         | Example                                  | Type      |
 |-------------|----------------------------------|-------------------------------------|------------------------------------------|-----------|
-| `playerId`  | required (if not send unitId)    | VideoLibrary player ID              | `'0151f985-fb1a-4f37-bb26-cfc62e43ec05'` | `string`  |
-| `unitId`    | required (if not send playerId)  | Monetize unit ID                    | `'AACBTwsZVANd9NlB'`                     | `string`  |
-| `vpaidMode` | optional                         | Vpaid wrapper; default: `false`.    | `true`                                   | `boolean` |
+| `unitId`    | required                         | Monetize unit ID                    | `'AACBTwsZVANd9NlB'`                     | `string`  |
+
+All other parameters should be sent inside openRTB request.
+
+### openRTB2 support
+
+{: .alert.alert-danger :}
+Starting with Prebid.js version 9.15 ShowHeores bidder adapter is requesting bids via openRTB protocol.  
+Publishers can use the `ortb2` method of setting [First Party Data](https://docs.prebid.org/features/firstPartyData.html). Bidder supports all first-party data fields: site, user, segments, and imp-level first-party data.
+
+### testing
+
+While developing or testing locally, you can request a test request by marking the openRTB as a test.
+To do that specifically for `showheroes-Bs` you can do:
+
+```javascript
+pbjs.setBidderConfig({
+  bidders: ['showheroesBs'],
+  config: {
+    ortb2: {
+      test:1
+    }
+  }
+})
+```
+
+Or, more easily you can mark the whole request as a test request by doing:
+
+```javascript
+pbjs.setConfig({config: {ortb2: {test: 1}}})
+```
+
+#### Outstream
+
+Example of adunit configuration for the outstream unit:
+
+```javascript
+
+var pbjs = pbjs || {};
+pbjs.que = pbjs.que || [];
+var adUnits = [
+    {
+        // Video adUnit
+        code: 'video-div-1',
+        mediaTypes: {
+            video: {
+                context: 'outstream',
+                playerSize: [640, 480],
+                mimes: ['video/mp4'],
+                playbackmethod: [2, 4, 6],
+                api: [1, 2, 4, 6],
+                protocols: [3, 4, 7, 8, 10],
+                placement: 1,
+                minduration: 0,
+                maxduration: 60,
+                startdelay: 0,
+                skip: 1
+            },
+        },
+        bids: [{
+            bidder: "showheroes-bs",
+            params: {
+                unitId: "AACBTwsZVANd9NlB",
```

Comment:
Developers will use these parameters to test their config.
Here some open questions coming to my mind:

What unit id are we using here ? 
Do we have a generic test unit created on our platform, that will be used during the integrations implementation?
Is our analytics tracking those test requests, or completely discarding it?

Reply:
As discussed privately, we now provide fake zid.

When using the preview mode, a test ad will be returned, even if the zid is not valid.
No tracking will be performed on analytics.

When the developer will remove the preview mode, the bidder will stop returning any bids.